eras/conway/impl/cardano-ledger-conway.cabal

```
@@ -166,6 +164,21 @@ library testlib
         text,
         small-steps >=1.1
 
+executable huddle-cddl
+    main-is:          Main.hs
+    hs-source-dirs:   huddle-cddl
+    other-modules:    Paths_cardano_ledger_conway
+    default-language: Haskell2010
+    ghc-options:
+        -Wall -Wcompat -Wincomplete-record-updates
+        -Wincomplete-uni-patterns -Wredundant-constraints -Wpartial-fields
+        -Wunused-packages -threaded -rtsopts -with-rtsopts=-N
+
+    build-depends:
+        base,
+        testlib,
+        cardano-ledger-binary:testlib
```

This is to ensure we don't try to build it with older `cardano-ledger-binary` that doesn't have `writeSpec`
```suggestion
        cardano-ledger-binary:testlib >= 1.3.4.0
```